central/productusage/csv/csv_handler.go

```diff
@@ -0,0 +1,80 @@
+package usagecsv
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/pkg/errors"
+	datastore "github.com/stackrox/rox/central/productusage/datastore/securedunits"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/csv"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/protoconv"
+)
+
+var (
+	zeroTime  = time.Time{}
+	csvHeader = csv.Row{"Timestamp", "Nodes", "CPU Units"}
+)
+
+func getSecuredUnitsConverter() csv.Converter[storage.SecuredUnits] {
+	record := make(csv.Row, 3)
+	return func(m *storage.SecuredUnits) csv.Row {
+		record[0] = protoconv.ConvertTimestampToTimeOrDefault(m.GetTimestamp(), zeroTime).UTC().Format(time.RFC3339)
+		record[1] = fmt.Sprint(m.GetNumNodes())
+		record[2] = fmt.Sprint(m.GetNumCpuUnits())
+		return record
+	}
+}
+
+// CSVHandler returns an HTTP handler function that serves usage data as CSV.
+func CSVHandler(ds datastore.DataStore) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		from, to, err := parseRequest(r)
+		if err != nil {
+			err = errox.InvalidArgs.New("bad CSV usage metrics request").CausedBy(err)
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+
+		csvWriter := csv.NewHTTPWriter(w,
+			"secured_units_usage.csv", getSecuredUnitsConverter(),
```

Comment:
A convenient thing about the diagnostic bundle that we might want to reuse - it includes a timestamp in the filename, so it's more unique and you can sort easier. Perhaps we should reuse this approach here?